src/BaseBallGame.js

```diff
@@ -0,0 +1,44 @@
+export default class BaseballGame {
+  play(computerInputNumbers, userInputNumbers){
+    let strikeCount = 0;
+    let ballCount = 0;
+
+    for(let i = 0 ; i < 3; i+=1){
+      let result = this.compare(computerInputNumbers,userInputNumbers[i], i);
+      if(result === 'strike'){
+        strikeCount+=1;
+      }
+      else if(result === 'ball'){
+        ballCount+=1;
+      }
+    }
+    return this.makeResult(strikeCount,ballCount);
+  }
+
+  compare(computerNumbers, userNumber, idx){
+    for(let i =0; i < computerNumbers.length ; i+=1){
+      if(computerNumbers[i] === userNumber && i === idx){
+        return 'strike';
+      }
+      if(computerNumbers[i] === userNumber && i !== idx){
+        return 'ball';
+      }
+    }
+    return 'Nothing';
+  }
+
+  makeResult(strikeCount, ballCount){
+    let output = '';
+    if(ballCount !== 0){
+      output += (ballCount.toString() + '볼 '); 
+    }
+    if(strikeCount !==0){
+      output += (strikeCount.toString() +'스트라이크')
+    }
+    if(output ==='')
+    {
+      output = '낫싱'
+    }
```

Comment:
사소하지만 코드 컨벤션을 지키려는 노력은 굉장히 중요해요. 
- 세미콜론이 없어요.
- brace의 위치가 일정하지 않아요

> 공통 요구사항
> [Airbnb 자바스크립트 코드 컨벤션](https://github.com/airbnb/javascript)을 지키면서 프로그래밍 한다.
```suggestion
    if(output ===''){
      output = '낫싱';
    }
```

나중에 어떠한 개발직군이든 집단내의 코드 품질을 보장하며 코드를 작성하는 것은 매우 중요한 일이에요! 그렇기 위해서 우리는 코드를 쓰는 규칙과 약속을 정하는데 그것을 우리는 **코드 컨벤션**이라고 부릅니다. 문서에 기재된 에어비앤비 코드 컨벤션도 그러한 것들 중 하나에요. 
하지만 실제로 에어비앤비 코드 컨벤션에 적힌 모든 것을 전부 읽고 외운 후에 코드를 작성하는 것은 우리가 컴퓨터가 아니라면 불가능합니다. 그러면 컴퓨터한테 이 코드가 약속을 지킨 코드인지 검사하게 할 수 있는 방법이 있지 않을까요? 한번 찾아보시고 적용해보시면 좋을 것 같습니다!

Reply:
제 나름 코드 컨벤션을 지키려고 해봤는데 실수가 있었던 것 같습니다. 다음에는 좀 더 노력해 보겠습니다!

찾아보니 검사를 할때는 ESlint를 많이 사용하고 추가로 prettier를 통해서 자동으로 코드를 수정한다고 합니다.
저는 아직 배우는 과정이니 검사를 위해 ESlint만 사용하는 것이 좋을까요?